components/scream/src/share/field/field.hpp

```diff
@@ -471,63 +471,58 @@ sync_to_dev () const {
 template<typename RealType>
 template<HostOrDevice HD>
 void Field<RealType>::
-set_value (const field_type& field_src) {
+deep_copy (const field_type& field_src) {
   const auto& layout     = get_header().get_identifier().get_layout();
   const auto& layout_src = field_src.get_header().get_identifier().get_layout();
   EKAT_REQUIRE_MSG(layout==layout_src,
        "ERROR: Unable to copy field " + field_src.get_header().get_identifier().name() + 
-          " to field " + get_header().get_identifier().name() + " during initialization.  Layouts don't match.");
+          " to field " + get_header().get_identifier().name() + ".  Layouts don't match.");
   const auto  rank = layout.rank();
   // Note: we can't just do a deep copy on get_view<HD>(), since this
```

Comment:
This comment has to be updated now. We could leave just the 1st sentence, and remove the other ones.

Reply:
oh yeah, of course it does.  Sorry for the lazy "response to reviewer comments" commit.